src/Command/SymlinksCommand.php

```diff
@@ -284,4 +281,16 @@ private function filterNestedPaths(Finder $finder, $prepend)
 
         return array_values($files);
     }
+
+    /**
+     * Returns the path relative to the root directory.
+     *
+     * @param string $path
+     *
+     * @return string
+     */
+    private function getRelativePath($path)
+    {
+        return str_replace(strtr($this->rootDir, '\\', '/').'/', '', strtr($path, '\\', '/'));
```

Comment:
Please don't try to fix things just by looking at them. The whole symlinking is not trivial, which is why we have thoroughly unit tested everything. And this method is definitely doing what it is supposed to to – which also includes returning an absolute path if `$path ` is not within `$this->rootDir`. 😉